user/hyfetch/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "hyfetch"
+pkgver = "1.4.11"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "bash",
+    "python-setuptools",
+    "python-typing_extensions",
```

Comment:
```suggestion
```
backport this https://github.com/hykilpikonna/hyfetch/commit/359226c26b065909f348084446ebad4fe0cc0680